packages/types/src/interfaces/index.ts

```diff
@@ -1,4 +1,4 @@
 // Auto-generated via `yarn polkadot-types-from-defs`, do not edit
 /* eslint-disable */
 
-export * from './types';
+export * from './types.js';
```

Comment:
Why did you change exports?

Reply:
These are auto generated after building code